ctags.lua

```diff
@@ -215,37 +215,64 @@ local function tag_cmd(tag, force)
 	end
 end
 
+local function gen_vis_menu(matches)
+	-- enforced max width of the desc field (file name) in the menu
+	local maxwidth = 40
+
+	local width = 0
+	for _, match in ipairs(matches) do
+		width = math.max(width, math.min(match.desc:len(), maxwidth))
```

Comment:
Let's maybe simplify this a little bit:
1. Remove `math.min()`, so that `width` is literally the longest width.
2. Use `math.min()` only once after the loop. We can even probably inline the `maxwidth` and just put the first comment near that line.